lib/src/common/polymer_descriptor.dart

```diff
@@ -202,7 +202,13 @@ JsObject _jsType(Type type) {
       return context['String'];
     case 'Map':
     case 'JsObject':
-    default:
       return context['Object'];
+    default:
+      // Store the type in the same manner as done in _addDartInstance so the
+      // type is retrieved in dartValue through _getDartInstance.
+      var value = new JsObject(context['Object']);
```

Comment:
Weird. It passes with pub test but if I run the individual file it fails. On the latest dev if it matters.

No problems in the dependent project.